Footer

ahmedidrissi opened the pull request.

AsmaHmida99 committed:
Update Footer

ahmedidrissi committed:
fix footer responsivity and change icons

ahmedidrissi committed:
Update required node version to pass Netlify checks

ahmedidrissi committed:
Replace ' with &apos;

Cristal32 reviewed (commented):
Looks good, the only thing I can maybe say and that's only my take on it, is to make it UM5 or UM5R instead of UMR5.

elbanas commented on src/components/Footer/index.tsx:
```diff
@@ -26,14 +26,69 @@ const Footer = () => {
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 24 24"
+                  >
+                    <path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2zm0 2v12h16V6H4zm8 6l8-5H4l8 5zm0 2L4 9v8h16v-8l-8 5z" />
```
this icon do not blend quietly with the other elements, and i think we can do better

elbanas replied:
suggestion :
remove `<path> ... </path>`  node and replace it with

`<g xmlns="http://www.w3.org/2000/svg">
<polyline fill="none" points="4 8.2 12 14.1 20 8.2" stroke="#959cb1" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></polyline>
<rect fill="none" height="14" rx="2" ry="2" stroke="#959cb1" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" width="18" x="3" y="6.5"></rect>
</g>`

sourced from : [https://www.svgrepo.com/svg/513836/mail](here)
license : CC

elbanas commented on src/components/Footer/index.tsx:
```diff
@@ -89,124 +146,158 @@ const Footer = () => {
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        strokeWidth="2"
+                        d="M3 12l2-2m0 0l7-7 7 7m-2 2v7a2 2 0 01-2 2H9a2 2 0 01-2-2v-7m10 0l-2 2m-6-2l-2 2"
```
same for this icon

replace `<path> ...</path>` with

`<path xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd" d="M11.3103 1.77586C11.6966 1.40805 12.3034 1.40805 12.6897 1.77586L20.6897 9.39491L23.1897 11.7759C23.5896 12.1567 23.605 12.7897 23.2241 13.1897C22.8433 13.5896 22.2103 13.605 21.8103 13.2241L21 12.4524V20C21 21.1046 20.1046 22 19 22H14H10H5C3.89543 22 3 21.1046 3 20V12.4524L2.18966 13.2241C1.78972 13.605 1.15675 13.5896 0.775862 13.1897C0.394976 12.7897 0.410414 12.1567 0.810345 11.7759L3.31034 9.39491L11.3103 1.77586ZM5 10.5476V20H9V15C9 13.3431 10.3431 12 12 12C13.6569 12 15 13.3431 15 15V20H19V10.5476L12 3.88095L5 10.5476ZM13 20V15C13 14.4477 12.5523 14 12 14C11.4477 14 11 14.4477 11 15V20H13Z" fill="#959cb1"></path>`

source : [https://www.svgrepo.com/svg/510001/home](link)
License : MIT

elbanas reviewed (changes requested):
everything looks great ,except 2 icons i mentioned in my review.
![Screenshot from 2024-09-04 09-29-20](https://github.com/user-attachments/assets/0949c6ce-4fba-42a9-a785-93a02a7f6d41)

@AsmaHmida99 please replace them (either with my suggestion or with your choice)

Cristal32 committed:
Replace email icons with requested ones. Replaced UMR5 with UM5R

Cristal32 reviewed (approved):
Updated the code with the requested changes:
- Replaced email and home icons with the new requested ones
- Changed UMR5 to UM5R

elbanas committed:
Fix warning : Invalid DOM property `stroke-linecap`. Did you mean `strokeLinecap`?
and other similar warnings

based on : https://stackoverflow.com/questions/66685543/reactjs-invalid-dom-property-error-on-svg-elements

elbanas commented:
fixed small warning before merging to master branch

![scrsht](https://github.com/user-attachments/assets/7a72503e-4805-4ef3-9cf2-87218a0664d0)

based on [https://stackoverflow.com/questions/66685543/reactjs-invalid-dom-property-error-on-svg-elements](url)
